doc/Makefile

```diff
@@ -53,6 +53,7 @@ clean:
 
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	jsdoc ../emperor/support_files/js/ -d $(BUILDDIR)/jsdoc -c ../jsdoc-config.json -r
```

Comment:
`jsdoc-config.json` is not committed in this PR. Also if that config file is part of the documentation, I would suggest putting it inside the `doc` directory instead of the base directory.